test/hisi_sec_test/test_hisi_sec.c

```diff
@@ -289,36 +291,32 @@ static int init_sigle_ctx_config(int type, int mode)
 {
```

Comment:
Single in init_single_ctx_config() means the single context. The original meaning is changed after multiple contexts are supported. You need to change the function name, too.

Reply:
Fixed.